### PR TITLE
Add cavm saiserver

### PR DESCRIPTION
**Pull request by NStetskovych**
Added cavium saiserver for PTF testing

### CODE REVIEW

File: platform/mellanox/sdk.mk [hunk 1]
```diff
@@ -1,4 +1,4 @@
-MLNX_SDK_BASE_URL = https://github.com/Mellanox/SAI-Implementation/raw/sonic/sdk
```
Comment (taoyl-ms) [inline platform/mellanox/sdk.mk]: What's the purpose of this change?

Comment (taoyl-ms) [inline platform/mellanox/sdk.mk]: What's the purpose of this change?

**Comment by stcheng** [comment]
could you update the pull request against the current master?

File: platform/mellanox/sdk.mk (hunk 1 above)
Comment (stcheng) [inline platform/mellanox/sdk.mk]: the above is introduced in other commits, not this one.
